.github/workflows/integration.yml

```diff
@@ -46,10 +46,25 @@ jobs:
       int_tests: true
     secrets: inherit
 
+  detect-user:
+    name: Detect user team membership
+    runs-on: ubuntu-latest
+    outputs:
+      IS_TEAM_MEMBER: ${{ steps.author.outputs.isTeamMember }}
+    steps:
+      - name: Get PR author
+        if: github.event_name == 'pull_request'
+        id: author
+        uses: tspascoal/get-user-teams-membership@v2
+        with:
+          username: ${{ github.actor }}
+          team: 'Vector'
+          GITHUB_TOKEN: ${{ secrets.GH_PAT_ORG }}
```

Comment:
I just realized that I don't think this will work on contributor PRs which don't have access to secrets. We could use the existence of the secret as the check for the `pull_request` target.